java/mykola-melnyk/src/com/zoolatech/lecture7/tasks/_2/Test_Task2.java

```diff
@@ -0,0 +1,31 @@
+package com.zoolatech.lecture7.tasks._2;
+
+public class Test_Task2 {
+    public static void main(String[] args) throws InterruptedException {
+        Calc_v_3 calc3 = new Calc_v_3(1);
+        Calc_v_2 calc2 = new Calc_v_2(1);
+        Calc_v_1 calc1 = new Calc_v_1(1);
+        Test_Task2.testCalc(calc3);
```

Comment:
you don't need to specify the class name to access a static method in the same class

Reply:
Removed redundant class names, thank you